src/main/java/utils/FormUrlEncodedParsingStrategy.java

```diff
@@ -0,0 +1,28 @@
+package utils;
+
+import static constant.Constant.*;
+
+import java.util.Arrays;
+import java.util.Map;
+import java.util.stream.Collectors;
+
+public class FormUrlEncodedParsingStrategy implements HttpBodyParsingStrategy {
+    @Override
+    public Map<String, String> parse(String body) {
+        if (body.isBlank()) {
+            return Map.of();
```

Comment:
아래와 같이 비어있는 Collection에 대해서는 `Collections.emptyMap();`를 활용하는 것이 좋을 것 같아요. 🙂
```suggestion
            Collections.emptyMap();
```